docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+
+* Line 1 helps `Reach` to decide how to compile or run the DApp.
+Without it, you will get an error.
+
+* Line 3 declares a `commonInteract` `{!rsh} object`. 
+This will hold properties that are common to those participating in this transaction `seller` and `buyer`.
+
+* Line 5 declares a `sellerInteract` `{!rsh} object`.
+ The `seller` will be given access to all the properties that go into the `sellerInteract`.
+
+* Line 7 declares a `buyerInteract` `{!rsh} object`.
+ It contains everything that the `buyer` will be able to do during this transaction.
+
+* Lines 9 and 19 are the start and end of the main export from the program. 
+Only the code within this block is compiled by the compiler. 
+
+* Lines 10 to 13 defines a `{!rsh} Participant` (i.e. `Seller`) and gives the `{!rsh} Participant` access to the `commonInteract` and `sellerInteract` properties.
+
+* Lines 14 to 17 defines another `{!rsh} Participant` (i.e. `Buyer`) and gives the `{!rsh} Participant` access to the `commonInteract` and `buyerInteract` properties.
+
+* Line 18 is used to start executing the program.
+Without it, the program will never start.
+
+
+### {#tut-mkt-starter-frontend} Frontend
+
+
+Next, work on the `index.mjs` file. Type the code below:
+
+```reach
+load: /examples/mkt-1-starter/index.mjs
+```
+
+* Line 1 imports `{!rsh} loadStdlib` from the Reach standard library.
+
+* Line 2 imports the `./build/index.main.mjs` file as the backend.
+This file doesn't exist yet.
+It will be automatically generated when you run `{!cmd} ./reach run` in your terminal.
+
+* Line 3 calls the `loadStdlib` function that was imported on line 1 and the output is stored as `stdlib`.
+
+* Line 5: The `Seller` sets up a contract for the program.
+
+* Line 6: The `Buyer` attaches to the contract.
+
+* Lines 8 to 17 is an asynchronous block that is used to execute the whole of the frontend code.
+
+* Lines 9 to 12 execute the `Seller`'s contract on the `Seller`. 
+The `Seller` is also given access to `commonInteract` and `sellerInteract` properties. 
+
+* Lines 13 to 16 execute the `Buyer`'s contract on the `Buyer`. 
+But this time around, the `Buyer` is given access to the `commonInteract` and `buyerInteract` properties.
+
+The code that you have just written forms the basis for the DApp that is to be built.
+
+Notice that `commonInteract`, `buyerInteract`, and `sellerInteract` are similar in both the backend and frontend files. 
+
+This is so because `Reach` needs to be sure of how the variables are to be used before being used. 
+The program will not be executed if those variables are not used the way they were declared.
+So these variables are declared at the backend before being defined in the frontend. 
+
+This might feel like a lot of work especially if you are coming from a language like JavaScript that is weakly typed.
+But we will rather use an application with strong security than one with weak security especially when it has to do with money and trust.
```

Comment:
However, it is better to use a language with stronger security, especially when dealing with money and trust.